src/scraper/spider_manager.py

```diff
@@ -0,0 +1,81 @@
+# external imports
+import scrapy
+from scrapy.crawler import CrawlerProcess
+import scrapy.signals
+
+# Project imports
+import scraper.scrapy_settings as settings
+
+# Python imports
+from typing import List
+
+
+class SpiderManager:
+    """
+        Utility class to perform common operations in 
+        Spider classes
+    """
+
+    process = CrawlerProcess(settings.CRAWLER_SETTINGS)
+
+    def __init__(self, spider) -> None:
+        self.spider = spider
+        pass
```

Comment:
why is there a pass here?

Reply:
Oh, it shouldn't be there, I just forgot delete it. Thanks for notice.